tests/test_parser.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+
+from io import BytesIO
+from sys import maxint
+from werkzeug.datastructures import MultiDict
+from werkzeug.exceptions import RequestEntityTooLarge
+from werkzeug.formparser import FormDataParser, default_stream_factory
+
+
+class TestFormDataParser(object):
+    """
+    Testcases for `FormDataParser`
+    """
+
+    @pytest.mark.parametrize('size,expected', [
+        (-1, BytesIO),
+        (0, BytesIO),
+        (1024 * 500, BytesIO),
+        (1024 * 501, file),
+        (maxint, file),
+    ])
+    def test_default_stream_factory(self, size, expected):
+        assert isinstance(default_stream_factory(size, '', ''), expected)
```

Comment:
`default_stream_factory` takes three parameters but only uses the first. I thought about refactoring but I wanted to get a maintainers opinion before I started.